routes.js

```diff
@@ -10,9 +10,14 @@ router.use(express.json());
 router.use(express.urlencoded({ extended: true }));
 
 router.use("/login", express.static("public/login.html"));
+router.use("/register", express.static("public/register.html"));
 
 router.post("/authenticate", checkUserCredentials, setUserJwt);
 
+router.post("/register", (request, response) => {
+  response.send("User registered");
```

Comment:
A common way of acknowledging a successful action is to respond with a 2xx response code. In this case, we're registering a user. i.e. _creating_ a user, and so perhaps a 201 response code would be appropriate.